composer.json

```diff
@@ -43,7 +43,7 @@
         "vendor-dir": "core/vendor"
     },
     "require": {
-        "php": ">=7.0",
+        "php": ">=7.1",
```

Comment:
Isn't it a better idea to have a separate commit/pr for the minimum required version bump? If 7.1 is the minimum required version we also have to change the version checks (for example in `manager/index.php`).

Reply:
I had to do this so it passed the checks. 7.0 will no longer pass the unit tests.

Reply:
Then we need to officially bump the minimum PHP version required for MODX 3.x to 7.1.